lib/ecto/changeset.ex

```diff
@@ -1275,6 +1276,62 @@ defmodule Ecto.Changeset do
 
   ## Validations
 
+    @doc """
+  Returns a keyword list of the validations for this changeset.
+
+  The keys in the list are the names of fields, and the values are a
+  validation associated with the field. A field may occur multiple
+  times in the list.
+
+  A quick example:
+
+      iex> %Post{}    |>
+      ...> change()   |>
+      ...> validate_format(:title, ~r/^\w+:\s/,
+      ...>      message: "must start with a topic")  |>
+      ...> validate_length(:title, max: 100)
+      ...> validation_specs()
```

Comment:
missing a pipe

Reply:
José:

I'm a little wary of looking like a getter (even though the function is), simply because it sends the wrong message. How about `all_validations` or `validation_list`?